crates/biome_module_graph/src/js_module_info.rs

```diff
@@ -168,6 +169,22 @@ pub enum JsExport {
     ReexportType(JsReexport),
 }
 
+impl JsExport {
+    pub fn as_own_export(&self) -> Option<&JsOwnExport> {
+        match self {
+            JsExport::Own(own_export) | JsExport::OwnType(own_export) => Some(own_export),
+            JsExport::Reexport(_) | JsExport::ReexportType(_) => None,
+        }
+    }
+
+    pub fn as_own_export_mut(&mut self) -> Option<&mut JsOwnExport> {
+        match self {
+            JsExport::Own(own_export) | JsExport::OwnType(own_export) => Some(own_export),
+            JsExport::Reexport(_) | JsExport::ReexportType(_) => None,
+        }
+    }
```

Comment:
What's the difference between these two methods? Their implementation seems the same

Reply:
Only difference is the mutability.

Reply:
Yeah, that I know, but why the first one doesn't return `&own_export` for example? Just trying to understand the magic behind it 😄

Reply:
Oh, it's because the pattern matching inside `match self { ... }` automatically propagates the mutability of the references.